.eslintrc.js

```diff
@@ -1,45 +1,18 @@
 module.exports = {
-  env: {
-    browser: true,
-    es6: true,
-  },
   extends: [
-    'standard',
-    'standard-react',
-    'prettier/react',
+    'react-app',
+    'prettier/@typescript-eslint',
     'plugin:prettier/recommended',
   ],
-  parser: 'babel-eslint',
-  parserOptions: {
-    ecmaFeatures: {
-      experimentalObjectRestSpread: true,
-      jsx: true,
-    },
-    sourceType: 'module',
-  },
-  plugins: ['prettier', 'react', 'react-hooks'],
+  plugins: ['react-hooks'],
   rules: {
-    'react/prop-types': 'warn',
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'error',
-    'import/no-unresolved': [
```

Comment:
I was looking at create-react-app, and didn't see the import plugin. This is a particular one that I would like to keep to help vim (do you get import checks for free in VSC or other modern editors?).

Reply:
Import resolution should be handled by typescript and the eslint plugin so I think this would be redundant now. Does vim give you trouble with it?

Reply:
I haven't set up much of a TS environment with vim. @bpierre will know better, but it's likely we can assume we don't need it then!

Reply:
Yep, the TS server takes care of it, so the import plugin isn't really needed. 👍